homework/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <title>HYF GitHub Repo Browser</title>
+  <link rel="stylesheet" href="./main.css">
+</head>
+
+<body>
+  <div id="root">
+    <header class="header">
+      <p>HYF Repositories</p>
+      <select id="repo-select" class="repo-selector">
+        <option>Please choose a repo</option>
+      </select>
+      <img id="logo" src="hyfrepo.png">
+    </header>
+    <div id="container">
+      <div class="left-div whiteframe repo-properties">
+        <div><label>Repository:</label> <span id="repo-name"></span></div>
+        <div><label>Description:</label> <span id="repo-description"></span></div>
+        <div><label>Forks:</label> <span id="repo-forks"></span></div>
+        <div><label>Updated:</label> <span id="repo-updated"></span></div>
+      </div>
+      <div class="right-div whiteframe">
```

Comment:
You could consider only displaying this once the list of contributors is loaded; displaying the loading indicator *instead of* this entire div.